### PR TITLE
Update AsyncUDP library.json

### LINKED ISSUE #186: ESP32
Hi,
    Please anyone tell me how to run BLE and WiFi in ESP32 simultaneously
Thanks

### PR DESCRIPTION
**Pull request by andig**
Fix https://github.com/espressif/arduino-esp32/issues/2836 and https://github.com/platformio/platform-espressif32/issues/186

### CODE REVIEW

**Comment by andig** [comment]
ping @me-no-dev does this make sense for you?

File: libraries/AsyncUDP/library.json [hunk 1]
```diff
@@ -1,17 +1,15 @@
...
-  {
-    "type": "git",
-    "url": "https://github.com/me-no-dev/ESPAsyncUDP.git"
-  },
-  "frameworks": "arduino",
-  "platforms":"espressif"
-}
+  "frameworks": [
+    "arduino"
+  ],
```
Comment (atanisoft) [inline libraries/AsyncUDP/library.json]: changing frameworks to a list is not necessary

File: libraries/AsyncUDP/library.json [hunk 2]
```diff
@@ -1,17 +1,15 @@
...
+  "platforms": [
+    "espressif32"
```
Comment (atanisoft) [inline libraries/AsyncUDP/library.json]: by changing this to espressif32 you are desupporting the esp8266, you will need to add "espressif" as a second element in this list.

**Comment by andig** [comment]
Happy to make the requested changes. 

As for the platform I was under the impression that this repo is for esp32 *only* and esp8266 i supported at https://github.com/me-no-dev/ESPAsyncUDP.git? Are you actually suggesting that I should open this PR upstream?

**Comment by atanisoft** [comment]
You are correct that this is only for the esp32, I missed which repo this was on and removed that comment after checking further. This version should only be for espressif32.

**Comment by andig** [comment]
Looking at the other embedded libraries- should we maybe drop the library.json altogether?

**Comment by atanisoft** [comment]
That is an option but it would need to be tested and confirm that PIO won't complain about it being missing.

**Comment by andig** [comment]
> That is an option but it would need to be tested and confirm that PIO won't complain about it being missing.

Just verified that my PIO project compiles without it present. No other included library does have this file either ;)

**Comment by atanisoft** [comment]
Then I'd suggest drop it from here :)

**Comment by andig** [comment]
Pr is already updated- up to you :)

**Comment by me-no-dev** [comment]
Hey @andig :) Long time no see :D

**Comment by andig** [comment]
> Hey @andig :) Long time no see :D

So true- hope you're doing well!

**Pull request merged**